onair/src/run_scripts/execution_engine.py

```diff
@@ -15,15 +15,14 @@
 import configparser
 import importlib
 import ast
-import shutil
-from shutil import copytree
+from shutil import copytree, rmtree
```

Comment:
This is optional and I can remove/uncommit. I believe I added this because I use Python 3.12 on my computer and the original method of rmtree is no longer supported on Python 3.12.

Reply:
Understood. No need to remove.